docs/whatsnew/6.0.rst

```diff
@@ -129,6 +129,35 @@ IERS data can now install the latest version of the
 download the package manually and transfer it to a computer that has no
 public internet connection.
 
+Parameter as a :func:`~dataclasses.dataclass`
+----------------------------
```

Comment:
The title underline being too short is breaking the documentation build, so while we are editing this anyways we might as well add another hyperlink.
```suggestion
:class:`~astropy.cosmology.Parameter` as a :func:`~dataclasses.dataclass`
-------------------------------------------------------------------------
```

Reply:
Done!